omise/omise.php

```diff
@@ -40,4 +40,100 @@ public function __construct()
     }
+
+    public function getContent()
+    {
+        $fields_form[0]['form'] = array(
+            'legend' => array(
+                'title' => $this->l('Settings')
+            ),
+            'input'  => array(
+                array(
+                    'type'     => 'switch',
+                    'label'    => $this->l('Enable/Disable'),
+                    'name'     => 'module',
+                    'is_bool'  => true,
+                    'desc'     => $this->l('Enable Omise Payment Module.'),
+                    'values'   => array(
+                        array(
+                            'id'    => 'module_enabled',
+                            'value' => 1,
+                            'label' => 'Enabled'
+                        ),
+                        array(
+                            'id'    => 'module_disabled',
+                            'value' => 0,
+                            'label' => 'Disabled'
+                        )
+                    )
+                ),
+                array(
+                    'type'     => 'switch',
+                    'label'    => $this->l('Sandbox'),
+                    'name'     => 'sandbox',
+                    'is_bool'  => true,
+                    'desc'     => $this->l('Enabling sandbox means that all your transactions will be in TEST mode.'),
+                    'values'   => array(
+                        array(
+                            'id'    => 'sandbox_on',
+                            'value' => 1,
+                            'label' => 'Enabled'
+                        ),
+                        array(
+                            'id'    => 'sandbox_off',
+                            'value' => 0,
+                            'label' => 'Disabled'
+                        )
+                    )
+                ),
+                array(
+                    'type'     => 'text',
+                    'label'    => $this->l('Public key for test'),
+                    'name'     => 'publicKeyForTest',
+                    'required' => false,
+                    'desc'     => 'The "Test" mode public key can be found in Omise Dashboard.'
+                ),
+                array(
+                    'type'     => 'text',
+                    'label'    => $this->l('Secret key for test'),
+                    'name'     => 'secretKeyForTest',
+                    'required' => false,
+                    'desc'     => 'The "Test" mode secret key can be found in Omise Dashboard.'
+                ),
+                array(
+                    'type'     => 'text',
+                    'label'    => $this->l('Public key for live'),
+                    'name'     => 'publicKeyForLive',
+                    'required' => false,
+                    'desc'     => 'The "Live" mode public key can be found in Omise Dashboard.'
+                ),
+                array(
+                    'type'     => 'text',
```

Comment:
I'm talking about https://github.com/omise/omise-prestashop/pull/5#r79105766 , permission for user group, sounds nice.